standard/structs.md

```diff
@@ -38,9 +38,12 @@ struct_modifier
     | 'protected'
     | 'internal'
     | 'private'
+    | unsafe_modifier   // unsafe code support
     ;
 ```
 
+*unsafe_modifier* is defined in [§23.2](unsafe-code.md#232-unsafe-contexts).
```

Comment:
Another case of the "*x* is defined in §clause" pattern as opposed to "*x* (§clause) is only available in unsafe code[§23]" one.